src/validation.cpp

```diff
@@ -1245,7 +1367,7 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
     // because it's unnecessary. Also, CPFP carve out can increase the limit for individual
     // transactions, but this exemption is not extended to packages in CheckPackageLimits().
     std::string err_string;
-    if (txns.size() > 1 && !PackageMempoolChecks(txns, package_state)) {
+    if (txns.size() > 1 && !PackageMempoolChecks(args, txns, workspaces, package_state)) {
```

Comment:
(just noting to remember) This lets through un-paid-for RBFs for single tx packages